Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -6,57 +6,65 @@
   - [1. Get a copy of the files (code and tutorial app)](#1-get-a-copy-of-the-files-code-and-tutorial-app)
   - [2. Install Edge Bridge using dependency manager (Swift Package Manager)](#2-install-edge-bridge-using-dependency-manager-swift-package-manager)
-  - [3. Update tutorial app code to enable Edge Bridge functionality](#3-update-tutorial-app-code-to-enable-edge-bridge-functionality)
+  - [3. Update tutorial app to enable Edge Bridge functionality](#3-update-tutorial-app-to-enable-edge-bridge-functionality)
   - [4. Run app](#4-run-app)
   - [5. `trackAction`/`trackState` implementation examples](#5-trackactiontrackstate-implementation-examples)
-- [Initial validation with Assurance](#initial-validation-with-assurance)
+- [Validation with Assurance](#validation-with-assurance)
   - [1. Set up the Assurance session](#1-set-up-the-assurance-session)
   - [2. Connect the app to the Assurance session](#2-connect-the-app-to-the-assurance-session)
-  - [3. Event transactions view - check for EdgeBridge events](#3-event-transactions-view---check-for-edgebridge-events)
-- [Data Prep mapping](#data-prep-mapping)
-- [Final validation using Assurance](#final-validation-using-assurance)
+  - [3. Event transactions view - check for Edge Bridge events](#3-event-transactions-view---check-for-edge-bridge-events)
+- [Optional - Data Prep for Data Collection mapping](#optional---data-prep-for-data-collection-mapping)
 
 ## Overview
-This hands-on tutorial provides end-to-end instructions on how to migrate from sending data to Analytics to sending data to the Edge Network using the Edge Bridge mobile extension.
+This tutorial covers how to use Edge Bridge as a drop-in solution for migrating from an existing Analytics implementation to sending data via the Edge Network to Analytics.
 
 ```mermaid
-graph LR;
-    step1(1<br/>Existing Adobe Analytics app) -->
+graph TD;
+    step1(1<br/>Existing app using Adobe Analytics) -->
     step2(2<br/>Adobe Experience Platform<br/>Update server-side configuration) --> 
-    step3(3<br/>Edge Bridge<br/>Send event data to the Edge Network & Analytics) --> 
-    step4(4<br/>Assurance<br/>Verify event data formats) -->
-    step5(5<br/>Data mapper<br/>Map data to XDM - Edge network data format) -->
-    step6(6<br/>Assurance<br/>Verify trackAction/trackState to XDM conversion)
+    step3(3<br/>Edge Bridge<br/>Update app to send event data via Edge Network to Analytics) --> 
+    step4(4<br/>Assurance<br/>Verify implementation) -. "Optionally" .-> 
+    step5("5<br/>Data mapper<br/>Map event data to Experience Data Model (XDM) format") -->
+    step6(6<br/>Assurance<br/>Verify event data to XDM conversion);
+
+style step1 fill:#EB1000,stroke:#333,stroke-width:2px,color:#fff
+style step2 fill:#EB1000,stroke:#333,stroke-width:2px,color:#fff
+style step3 fill:#EB1000,stroke:#333,stroke-width:2px,color:#fff
+style step4 fill:#EB1000,stroke:#333,stroke-width:2px,color:#fff
```

Comment:
I am thinking we can drop the optional steps from this tutorial doc, already covered in the other one